### DISCUSSION TITLE
Screenshot button change?

### DISCUSSION THREAD

**Discussion by LaserAJF** [ChristopherKlay/StadiaEnhanced#210] [Ideas & Requests]
I play Stadia on a chromebook, and unless I use a controller, there is no way to take a screenshot or clip. Would it be possible for Stadia enhanced to be able to change my screenshot key or or put a screenshot/clip recordor in the stadia menu?

**Comment by ChristopherKlay** [comment]
I heard there's issues with it in recent versions, but normally you should be able to use the function key, together with the number key row, to access F1-F12. `Fn` + `2` would equal F2, for example.

I'll look into emulating the clip/screenshot function for UI access (via mouse) as well, however (:
